nx/include/switch/services/hidsys.h

```diff
@@ -11,19 +11,19 @@
 
 /// Mini Cycle struct for \ref HidsysNotificationLedPattern.
 typedef struct {
-    u8 ledIntensity;      ///< Mini Cycle X LED Intensity.
-    u8 fadingTransition;  ///< Fading Transition Duration to Mini Cycle X (Uses PWM). Value is a Multiplier of HidsysNotificationLedPattern::globalMiniCycleDuration.
-    u8 ledDuration;       ///< LED Duration Multiplier of Mini Cycle X. x0 = x1 = x1
+    u8 ledIntensity;        ///< Mini Cycle X LED Intensity.
+    u8 transitionSteps;     ///< Fading Transition Steps to Mini Cycle X (Uses PWM). Value 0x0: Instant. Each step duration is based on HidsysNotificationLedPattern::globalMiniCycleDuration.
+    u8 finalStepDuration;   ///< Final Step Duration Multiplier of Mini Cycle X. Value 0x0: 12.5ms, 0x1 - xF: 1x - 15x. Value is a Multiplier of HidsysNotificationLedPattern::globalMiniCycleDuration.
     u8 pad;
 } HidsysNotificationLedPatternCycle;
 
 /// Structure for \ref hidsysSetNotificationLedPattern.
 /// See also: https://switchbrew.org/wiki/HID_services#NotificationLedPattern
 /// Only the low 4bits of each used byte in this struct is used.
 typedef struct {
-    u8 globalMiniCycleDuration;                         ///< Global Mini Cycle Duration. 8ms - 175ms. Value 0x0 = 0ms/OFF.
-    u8 totalMiniCycles;                                 ///< Number of Mini Cycles. 1-15. If number of cycles is > 0 then x0 = x1
-    u8 totalFullCycles;                                 ///< Number of Full Cycles. 1-15. Value 0x0 is repeat forever, but if globalMiniCycleDuration is set to 0x0, it does the 1st Mini Cycle and then the LED stays on with startIntensity.
+    u8 globalMiniCycleDuration;                         ///< Mini Cycle Base Duration. Value 0x1-0xF: 12.5ms - 187.5ms. Value 0x0 = 0ms/OFF.
```

Comment:
baseMiniCycleDuration?

Reply:
(update example pr that afterwards)

Reply:
I thought about that and didn't change it because the original also makes sense.
I'll push a fix to both

Reply:
done